other/js/password.txt

```diff
@@ -1 +0,0 @@
-passpass12
```

Comment:
Squash this commit (e5e0e5d5686384820a488e0f5fa8e40d942b6c28) with 0e1824c0ff64e008e0cd24de5f1a02f1de6e5394 please